src/selectors/resourceSelectors.js

```diff
@@ -0,0 +1,101 @@
+import { createSelector } from 'reselect';
+
+const getAllResources = state => state.filterMarkers.allResources;
+const getResourceType = state => state.filterMarkers.resourceType;
+const filterTags = state => state.filterMarkers.filterTags;
+const filterEntry = state => state.filterMarkers.filterEntry;
+
+const tagMapping = {
```

Comment:
Make a comment about this being a mapping from UI labels to the backend types

Reply:
fixed